Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -10275,6 +10282,7 @@ void WebPageProxy::resetState(ResetStateReason resetStateReason)
 
 #if ENABLE(WEB_AUTHN)
     protectedWebsiteDataStore()->authenticatorManager().cancelRequest(internals().webPageID, std::nullopt);
+    // FIXME: Implement equivelent Digital Credential Manager https://webkit.org/b/277850
```

Comment:
typo: "equivelent"
Also, WebKit comments should be sentences and end with a period.